usr/src/cmd/fs.d/nfs/mountd/mountd.c

```diff
@@ -3104,18 +3103,23 @@ check_sharetab()
 	}
 
 	/*
-	 * Remember the mod time, then after getting the
-	 * write lock check again.  If another thread
-	 * already did the update, then there's no
-	 * work to do.
+	 * Now that we have the lock, reverify that no one else beat us to the
```

Comment:
Please move this comment _after_ the `rw_wrlock()` call below.

Reply:
Thanks.  Yes, the comment does apply to the code after the rw_wrlock() call.